graylog2-web-interface/yarn.lock

```diff
@@ -5193,7 +5193,7 @@ clean-css@^5.1.5:
   dependencies:
     source-map "~0.6.0"
 
-clipboard@*, clipboard@^2.0.0:
+clipboard@*:
```

Comment:
Do you know why there is still an entry in the lockfile after the dependency has been removed?

Reply:
There was still a reference to ` @types/clipboard` that was causing this; I removed it, too.